.github/workflows/generate-goldens.yml

```diff
@@ -0,0 +1,37 @@
+# GitHub Action that generates screenshots for integration tests. These can be
+# downloaded and added to your PR.
+name: Artifacts
+
+on:
+  workflow_dispatch:
```

Comment:
I haven't yet been able to use it since I think the workflow needs to be merged to `main`. I was thinking once this is merged we can remove the pull and push request trigger (although the size of the artifact is currently tiny).